erizo_controller/erizoClient/src/Room.js

```diff
@@ -587,6 +587,14 @@ Erizo.Room = (specInput) => {
 
     options.simulcast = options.simulcast || false;
 
+    if (stream.videoSize && stream.videoSize[0] > 640 && options.simulcast &&
+      options.simulcast.numSpatialLayers <= 2) {
+      L.Logger.warning('Trying to publish an HD stream with only 2 quality layers. Due to a Chrome ' +
```

Comment:
a warning might be too high cause it's a bug in Chrome, that might be fixed at some point. I agree with the log, but I'd rather use L.logger.debug